week2/assignment/index.html

```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css" />
+    <link rel="stylesheet" href="./css/reset.css" />
+    <link rel="stylesheet" href="./css/style.css" />
+    <link rel="stylesheet" href="./css/modal.css" />
+    <title>week2</title>
+</head>
+
+<body>
+    <header>
+        <i class="fa-solid fa-bars"></i>
+        <h1>Wave Web</h1>
+        <h3>🌊 35th AND SOPT 김가현 🌊</h3>
+    </header>
+
+    <main>
+        <section class="filter-container">
+            <div class="filter full">
+                <label for="name">이름</label>
+                <input id="name" type="text" placeholder="이름을 입력하세요" />
+            </div>
+
+            <div class="filter full">
+                <label for="englishName">영문이름</label>
+                <input id="englishName" type="text" placeholder="영문이름을 입력하세요" />
+            </div>
+
+            <div class="filter one-third">
+                <label for="github">GitHub</label>
+                <input id="github" type="text" placeholder="Github ID를 입력하세요" />
+            </div>
+
+            <div class="filter one-third">
+                <label for="gender">성별</label>
+                <select id="gender">
+                    <!-- 기본 선택지를 설정할 때 빈 값으로 설정되도록 value=""를 해줘야 한다!!! -->
+                    <option disabled hidden selected value="">성별 선택</option>
+                    <option value="male">남자</option>
+                    <option value="female">여자</option>
+                </select>
+            </div>
+
+            <div class="filter one-third">
+                <label for="role">역할</label>
+                <select id="role">
+                    <option disabled hidden selected value="">OB / YB 선택</option>
+                    <option value="OB">OB</option>
+                    <option value="YB">YB</option>
+                </select>
+            </div>
+
+            <div class="filter half">
+                <label for="firstWeekGroup">1주차 금잔디조</label>
+                <input id="firstWeekGroup" type="number" placeholder="1주차 금잔디조를 입력하세요 (1~9)" />
+            </div>
+
+            <div class="filter half">
+                <label for="secondWeekGroup">2주차 금잔디조</label>
+                <input id="secondWeekGroup" type="number" placeholder="2주차 금잔디조를 입력하세요 (1~9)" />
+            </div>
+            <div class="filter full">
+                <div class="filter-btns">
+                    <button class="reset-btn">초기화</button>
+                    <button class="search-btn">검색</button>
```

Comment:
p2: `button` 태그의 `type`을 지정해주는 것은 어떨까요?
`button`의 `type`에는 3가지 값을 지정해 줄 수 있는데 각각 `submit`, `reset`, `button`입니다.
만약 아무런 값도 지정하지 않았다면 디폴트값은 `submit`이 됩니다.
그러니까` <button></button>` 과 `<button type="submit"></button>`은 같은 셈이죠.
더 자세한 내용은 첨부한 아티클 참고하면 좋을거 같아요!
https://nykim.work/96
```suggestion
                    <button type="reset" class="reset-btn">초기화</button>
                    <button type="button" class="search-btn">검색</button>
```

Reply:
오호 넵 ! 첨부해주신 아티클 잘 읽었습니다 ~
피드백 주신 부분 반영했습니다 ㅎㅎ